crates/ursa-rpc-service/src/api.rs

```diff
@@ -67,6 +71,12 @@ pub struct NetworkGetFileParams {
 }
 pub const NETWORK_GET_FILE: &str = "ursa_get_file";
 
+pub type EthSendTransaction = TransactionRequest;
```

Comment:
Is this type being used anywhere?

Reply:
It wasnt but i changed it to be used now to follow the pattern of the existing rpc endpoints